### PR TITLE
Add more documentation including limitations in the Scalar DB docs

### PR DESCRIPTION
**Pull request by feeblefakie**
https://scalar-labs.atlassian.net/browse/DLT-7909

### CODE REVIEW

**Review by brfrn169 — COMMENTED** [review]
Just left some comments. Please check them when you have time.

File: src/main/java/com/scalar/db/api/DistributedTransaction.java [hunk 1]
```diff
@@ -6,36 +6,132 @@
...
+  /**
+   * Returns the table name
+   *
+   * @return an {@code Optional} with the table name
+   */
   Optional<String> getTable();
 
+  /**
+   * Retrieves a result from the storage through a transaction with the specified {@link Get}
+   * command with a primary key and returns the result. The partition keys names and the clustering
```
[the remark this answers is not in the document]
Comment (feeblefakie) [reply src/main/java/com/scalar/db/api/DistributedTransaction.java:61]: Thank you!
As we discussed, it's a copy of StorageService so I'll update it for a transaction.
Also, I'll remove the second sentence from StorageService.

[the remark this answers is not in the document]
Comment (feeblefakie) [reply src/main/java/com/scalar/db/api/DistributedTransaction.java]: Fixed in 
07aa95c.

**Comment by brfrn169** [comment]
@feeblefakie Are you going to add the document to `ConsensusCommit` that the projections of Get and Scan will be cleared ?

**Comment by feeblefakie** [comment]
@brfrn169 Yes, I'll do it shortly. Thank you for reminding me!

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**